mycroft/util/settings_gui_generator.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2020 Mycroft AI Inc.
```

Comment:
Will this file be of use to other things than the gui? if not it may be better to keep it in the enclosure directory maybe?

Reply:
I think it should remain in utils, this helper class can be used by anyone to generate a synced data model from settingsmeta.json and settings.json

Reply:
Fair point.